client/state/sites/vouchers/actions.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	SITE_VOUCHERS_RECEIVE,
+	SITE_VOUCHERS_REQUEST,
+	SITE_VOUCHERS_REQUEST_SUCCESS,
+	SITE_VOUCHERS_REQUEST_FAILURE,
+} from 'state/action-types';
+
+/**
+ * Module vars
+ */
+const errorMessage = 'There was a problem fetching site vouchers. Please try again later or contact support.';
```

Comment:
Will this be seen by users? Maybe return an error code, and let the component translate the message.